AK/Time.h

```diff
@@ -56,24 +56,40 @@ constexpr int day_of_year(int year, unsigned month, int day)
 }
 
 // Month starts at 1. Month must be >= 1 and <= 12.
-int days_in_month(int year, unsigned month);
+int days_in_month(i64 year, unsigned month);
 
-constexpr int days_in_year(int year)
+constexpr int days_in_year(i64 year)
 {
     return 365 + (is_leap_year(year) ? 1 : 0);
 }
 
-constexpr int years_to_days_since_epoch(int year)
+constexpr i64 years_to_days_since_epoch(i64 year)
```

Comment:
This might have a bug for the year 2100. Before this change, `years_to_days_since_epoch` returned 47482. Now it returns 47481.

Reply:
After that is fixed, if you want a free commit, feel free to revert 032664332be6e1d42567165e9ffb623faf2b9a10 :)